src/ape/api/providers.py

```diff
@@ -508,6 +508,19 @@ def set_code(  # type: ignore[empty-body]
             code (:class:`~ape.types.ContractCode`): The new bytecode.
         """
 
+    @raises_not_implemented
+    def set_storage(  # type: ignore[empty-body]
+        self, address: AddressType, slot: int, value: HexBytes
```

Comment:
I thought I had remembered a virtual machine that didn't use storage slots, maybe it was Cairo?
Just wondering if this method would have a separate meaning then, if we should rename `slot`, stuff like that.

Reply:
Could just call it `key` I suppose, most VMs come with some sort of persistent key-value store for a stateful blockchain